### PR TITLE
added screenshooter extension

### LINKED ISSUE #97: ScreenShooter extension
A wrapper around the screen shooter optional Wayland extension.

[Here is the protocol](https://github.com/swaywm/wlroots/blob/master/protocol/screenshooter.xml) and [here is the relevant header file to wrap](https://github.com/swaywm/wlroots/blob/master/include/wlr/types/wlr_screenshooter.h).

### PR DESCRIPTION
**Pull request by SethBarberee**
closes #97 

Just like gamma control, I did the same process.

### CODE REVIEW

File: src/extensions/screenshooter.rs [hunk 1]
```diff
@@ -0,0 +1,27 @@
+use wayland_sys::server::wl_display as wl_server_display;
+use wlroots_sys::{wl_display, wlr_screenshooter_manager, wlr_screenshooter_manager_create, 
+    wlr_screenshooter_manager_destroy};
```
Comment (Timidger) [inline src/extensions/screenshooter.rs]: Make this line up with the previous line (e.g. the `w` in `wlr` should line up with `wl`. See the other files for examples, my editor does this automatically).

Comment (Timidger) [inline src/extensions/screenshooter.rs]: Looks like you have the wrong name of functions, they should not have the `manager` in the name.

**Comment by SethBarberee** [comment]
I need to push my updated stuff. I've fixed my naming on my local repo but just need to push it. I was merely updating the branch here with master with all the new stuff.

**Comment by SethBarberee** [comment]
Resolved all reviews with recent commit. Build complains about bindgen stuff.

**Comment by Timidger** [comment]
Thanks!

**Pull request merged**